COMMUNITY-GUIDELINES.MD

```diff
@@ -0,0 +1,19 @@
+#Community Guidelines
+
+We are a community based on trust, transparency and contribution. We have a trust by default policy and in order to keep it that way, we expect everyone to follow these community guidelines and act accordingly. 
+
+**Be honest** about who you are, what your collective is doing, why you are fundraising and what you want to do with those funds. Dishonesty is not only damaging to you but to all other collectives. Don’t be the one bringing mistrust into this community. 
+
+**The collective first.** Collectives are groups of people with a shared mission. People may come and go, but the mission remains. A successful collective is a collective that survives its initial founders.
+
+**Be inclusive.** The reason why we can do great things together, is because each of us bring a different perspective. Whoever you are, you have a unique perspective and unique skills that you can contribute to a collective.
+
+**Be respectful.** We have zero tolerance to trolling. Being a jerk will not get you a place in this community, it will turn people away. We don’t want noise in the system. 
+
+**Be mindful** Your backers will definitely want to hear about how you are going, get invitations to your activities, and stay connected. Please be respectful of their inboxes and their time and make sure they are always wanting more! 
```

Comment:
:D

P.
piamancini.com
t: @piamancini
s: piamancini
OpenCollective <https://opencollective.com>





On Fri, Apr 29, 2016 at 1:46 PM, Xavier Damman <notifications@github.com>
wrote:

> In COMMUNITY-GUIDELINES.MD
> <https://github.com/OpenCollective/OpenCollective/pull/58#discussion_r61642190>
> :
>
> > @@ -0,0 +1,19 @@
> > +#Community Guidelines
> > +
> > +We are a community based on trust, transparency and contribution. We have a trust by default policy and in order to keep it that way, we expect everyone to follow these community guidelines and act accordingly.
> > +
> > +**Be honest** about who you are, what your collective is doing, why you are fundraising and what you want to do with those funds. Dishonesty is not only damaging to you but to all other collectives. Don’t be the one bringing mistrust into this community.
> > +
> > +**The collective first.** Collectives are groups of people with a shared mission. People may come and go, but the mission remains. A successful collective is a collective that survives its initial founders.
> > +
> > +**Be inclusive.** The reason why we can do great things together, is because each of us bring a different perspective. Whoever you are, you have a unique perspective and unique skills that you can contribute to a collective.
> > +
> > +**Be respectful.** We have zero tolerance to trolling. Being a jerk will not get you a place in this community, it will turn people away. We don’t want noise in the system.
> > +
> > +**Be mindful** Your backers will definitely want to hear about how you are going, get invitations to your activities, and stay connected. Please be respectful of their inboxes and their time and make sure they are always wanting more!
>
> "Be mindful", yes I love that! 👍
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenCollective/OpenCollective/pull/58/files/9df4e5898046045521b5e09e765b25c23ba6f8bd#r61642190>
>